.github/workflows/cmake-builds.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04]

```

Comment:
Line 26 in build.yml now says `ubuntu-latest`, does that apply here too?

Reply:
cmake-builds.yml produces artifacts, so the idea here is to list out the operating systems for which artifacts are produced. There are two Ubuntu images in the GH image runner collection (one of which is "ubuntu-latest"), so I explicitly spelled out the versions.